kubeflow/common/iap-ingress/base/cert.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: networking.gke.io/v1beta1
+apiVersion: networking.gke.io/v1
```

Comment:
note, v1beta1 is deprecated, upgrade to v1 schema -- but the fields we used are compatible

Reply:
Thank you Yuan! Sounds good.